src/App.css

```diff
@@ -1,42 +1,18 @@
 #root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
   text-align: center;
 }
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
-}
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
-}
-.logo.react:hover {
-  filter: drop-shadow(0 0 2em #61dafbaa);
+ .app .bg-white {
+  background-color: white;
 }
 
-@keyframes logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.app .border-radius-right {
```

Comment:
The class name is not great. It sounds like a generic, utility class, but it doesn't tell us anything about the amount it sets for the border radius. Also it doesn't follow the Tailwind pattern.

Instead I would name this `search-input` and the button one below `search-button`. I think that makes it easier to understand.